make/Defs.gmk

```diff
@@ -174,9 +182,17 @@ else
 TIDY	= /usr/bin/tidy
 endif
 TOUCH 	= /usr/bin/touch
+ifeq ($(SYSTEM_UNAME), FreeBSD)
+UNZIP	= /usr/local/bin/unzip
```

Comment:
Once again, maybe use `which` instead? Unless you have two installations of unzip, and the one in /usr/bin is broken in some way. Then you might need to check for brokenness instead.